.babelrc

```diff
@@ -1,12 +1,14 @@
 {
   "presets": [
-    "es2015",
-    "react"
+   [ "@babel/preset-env", {
+     "modules": false
+   }],
+    "@babel/preset-react"
   ],
   "plugins": [
-    "transform-object-assign",
-    "transform-class-properties",
+    "@babel/plugin-transform-object-assign",
+    "@babel/plugin-proposal-class-properties",
     "transform-es2015-modules-commonjs",
-    "transform-object-rest-spread"
+    "@babel/plugin-proposal-object-rest-spread"
```

Comment:
https://babeljs.io/docs/en/v7-migration#switch-to-proposal-for-tc39-proposals-blog-2017-12-27-nearing-the-70-releasehtml-renames-proposal